src/modules/SdnDiag.Utilities/private/Invoke-WebRequestWithRetry.ps1

```diff
@@ -21,20 +23,23 @@ function Invoke-WebRequestWithRetry {
         [Parameter(Mandatory = $false)]
         [Switch] $UseBasicParsing,
 
-        [Parameter(Mandatory = $false)]
+        [Parameter(Mandatory = $true, ParameterSetName = 'Certificate')]
```

Comment:
Review `Mandatory` attribute

Reply:
This is correct when using ParameterSetName and helps define what is required. It enables unique properties to be required depending on the ParameterSetName that is used.